.circleci/config.yml

```diff
@@ -662,17 +688,20 @@ workflows:
             - build-nightly-buster-securedrop-workstation-config
       - build-nightly-buster-securedrop-workstation-grsec:
           requires:
-            - build-nightly-buster-securedrop-workstation-config
+            - build-nightly-buster-securedrop-keyring
       - build-nightly-bullseye-securedrop-log:
           requires:
-            - build-nightly-buster-securedrop-log
+            - build-nightly-buster-securedrop-workstation-grsec
+      - build-nightly-bullseye-securedrop-client:
+          requires:
+            - build-nightly-bullseye-securedrop-log
       - build-nightly-bullseye-securedrop-keyring:
           requires:
-            - build-nightly-buster-securedrop-keyring
+            - build-nightly-bullseye-securedrop-client
       - build-nightly-bullseye-securedrop-workstation-grsec:
           requires:
-            - build-nightly-buster-securedrop-workstation-grsec
+            - build-nightly-bullseye-securedrop-keyring
       # Cleaning old nightlies should always be the last step
       - clean-old-nightlies:
           requires:
-            - build-nightly-bullseye-securedrop-keyring
+            - build-nightly-bullseye-securedrop-workstation-grsec
```

Comment:
- `build-nightly-bullseye-securedrop-workstation-grsec` is indeed the last target at this point :white_check_mark: